coala-bears-requirements.txt

```diff
@@ -0,0 +1,32 @@
+# Use >= for development versions so that source builds always work
+coala>=0.7.0.dev20160525083409
```

Comment:
For now I have just copied the files from current Master branches. The 
weird result is that coala0.7.0 gets Installed first and later replaced by 
coala0.6.0

I'm not sure which one we actually want in the Docker Image


Am 13. Juni 2016 3:31:17 vorm. schrieb AbdealiJK <notifications@github.com>:

>> @@ -0,0 +1,32 @@
>> +# Use >= for development versions so that source builds always work
>> +coala>=0.7.0.dev20160525083409
>
> Do we want development versions ? OR is this meant for stable ones ?
>
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/coala-analyzer/docker-coala-base/pull/8/files/f8e37e129a7d4811e5ca9a7c1bbad28fec7c8eba#r66732330